README.md

```diff
@@ -17,6 +17,16 @@ The policy has three settings:
 * `run_as_group`:  Controls which primary group ID the containers are run with. As well as the group in the securityContext from PodSpec.
 * `supplemental_groups`: Controls which group IDs containers add.
 * `validate_container_image_configuration`: A boolean value that allows the policy to validate the `USER` directive in the container image configuration. The default value is `false`.
+> [!WARNING]  
+> When users enable the container image validation, the policy will perform
+> host capability call to access the container image metadata stored in the
+> registry. Which involves network access. This can slow down the policy
+> evaluation in a cold execution when this data is not store in the cache.
+> Therefore, the policy server can interrupt the evaluation if it takes too
+> long to complete. Pay attention to the `--policy-timeout` flag when starting
```

Comment:
let's point the user to https://docs.kubewarden.io/reference/policy-evaluation-timeout#configuration